cmd/influxd/upgrade/upgrade.go

```diff
@@ -149,6 +175,13 @@ func NewCommand() *cobra.Command {
 			Desc:    "path for boltdb database",
```

Comment:
It's too far away from the diff for me to comment on directly, but could you add a point `3` to the list at [this line](https://github.com/influxdata/influxdb/pull/19937/files#diff-e5c52e2be1a2771a344e5d0402468658d34a70fe9ba50453edb3b3b3e97a4eaeR149) describing the new behavior?

Reply:
I hope I got this correctly (wasn't sure where the link point to).

Reply:
Yup, that was it